<launch>
  <node name="squirrel_fixate_on_3D_point_node"
        pkg="squirrel_attention"
        type="fixate_on_3D_point_node"
        required="true" output="screen">

        <remap from="/pan_topic" to="/neck_pan_controller/rel_command"/>
        <remap from="/tilt_topic" to="/neck_tilt_controller/rel_command"/>

  </node>

</launch>
